system_test/common/aest_db_SUITE.erl

```diff
@@ -112,7 +123,7 @@ sc_leave_upgrade_reestablish_different_nodes_partial_upgrade(Cfg) ->
     sc_leave_upgrade_reestablish({{alice1, old}, {bob1, old}}, {{alice2, old}, {bob2, new}}, Cfg).
 
 sc_leave_upgrade_reestablish_different_nodes_partial_to_full_upgrade(Cfg) ->
-    sc_leave_upgrade_reestablish({{alice1, old}, {bob1, new}}, {{alice2, new}, {bob2, new}}, Cfg).
+    sc_leave_upgrade_reestablish({{alice1, new}, {bob1, old}}, {{alice2, new}, {bob2, new}}, Cfg).
```

Comment:
While the initiator is backwards compatible in regards of noise session messages, the responder is not. If we use a `new` (Lima) responder with an `old` (Fortuna) initiator, the new scheme of noise messages for [encoding the accept message](https://github.com/aeternity/aeternity/blob/master/apps/aechannel/src/aesc_codec.erl#L149-L201). This will crash.

On the other hand, if the responder is `old` and initiator is `new`, communication is backwards compatible. Since this test checks if `old` node can be upgraded to a `new` version and the channel is still able to read old state, the test is not impacted of this change.